ci/upload_to_owncloud.sh

```diff
@@ -50,6 +50,7 @@ main() {
     remote_path="$1"; shift
     [ -n "$1" ] || { usage; err "Missing local-path"; exit 1; }
     local_path="$1"; shift
+    OWNCLOUD_BROWSE_URL="https://ftp.essensium.com/owncloud/index.php/apps/files/?dir=$remote_path/$(basename "$local_path")"
```

Comment:
This can't be right - just below, the remote_path and local_path are added to it again. 

This definition should move down to where OWNCLOUD_BROWSE_URL is defined the first time.

Please use the public URL, so people who have no owncloud account can also click the link: `https://ftp.essensium.com/owncloud/index.php/s/ketl3eFhIgwcweZ?path=`. Note that this requires `$remote_path` to *not* include the certification/ bit.

Reply:
Right, I broke it when rebasing.
The thing is that the URL is not going to be the same for the different folders (the artifacts folder is not part of the prplmesh folder).
What I propose is to just drop this URL, and add links to the relevant directories in the README.md.
What do you think?

Reply:
I really prefer not to drop the URL since it is printed to stdout as part of the cron job and the idea is that when we get the mail we can simply click the link to get to the results.